pom.xml

```diff
@@ -233,6 +233,11 @@
                 <artifactId>cos-fleetshard-api</artifactId>
                 <version>${cos-fleetshard-api.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.bf2</groupId>
+                <artifactId>cos-fleetshard-api-deployment</artifactId>
+                <version>${cos-fleetshard-api.version}</version>
+            </dependency>
```

Comment:
Please remove this change again. #258 has a better fix, adding the dependency to the Quarkus plugin in the <plugin> section.

Reply:
Ok, sure!